packages/commerce/backend/src/app.ts

```diff
@@ -73,7 +80,7 @@ export class App {
       })
     })
 
-    router.use(this.errorHandler)
+    router.use(this.errorHandler.bind(this))
```

Comment:
can `errorHandler` be an arrow function?

Reply:
I'd rather keep this as it is. Making the `errorHandler` method an arrow function would prevent it from becoming private during build time, as it will be moved into the constructor to maintain the correct `this` reference.